docs/Onboarding/index.md

```diff
@@ -74,41 +74,50 @@
 1. On "Register your device" select Set-up and login with your MS account with TAP, and when registration is complete click "Done"
 1. Give Google Services permissions (location, diagnostic data etc...) and click on "Accept"
 1. Agree to End user License Agreement and click "Next"
-1. Configure Microsoft Authenticator app (login with company account and then register authenticator with company)
+1. Configure Microsoft Authenticator app (login with Microsoft company account of user that is going to be using device and then register authenticator with company)
 1. Login to Microsoft Defender (if already logged in with company account account should be present, just select it) and give it requested  permissions, wait for the scan to complete
 1. Open Outlook and Teams app, check if auto logged in (if already logged in with company account account should be present, just select it)
 
 ## Onboarding Device - Android Personally-owned devices with work profile
-
+1. Before continuing please login  with the desired Google personal account.
 1. Install Intune Company Portal from Google Play store, [https://play.google.com/store/apps/details?id=com.microsoft.windowsintune.companyportal](https://play.google.com/store/apps/details?id=com.microsoft.windowsintune.companyportal)
```

Comment:
add empty line above, as it was before.
delete your step 1
on step install intune company portal, , add at the end "Note: to use Google Play store, login with personal Google account is required"